<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MembraneTestControlArea</class>
 <widget class="QWidget" name="MembraneTestControlArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QTabWidget" name="full_tab">
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="controlWidget">
         <attribute name="title">
          <string>控制</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="stiWidget">
         <attribute name="title">
          <string>刺激</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="paramWidget">
         <attribute name="title">
          <string>参数</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CDataMonitorPanel" name="dataMonitorTableView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>195</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDataMonitorPanel</class>
   <extends>QTableView</extends>
   <header>CDataMonitorPanel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
